src/jupyterlabcontroller/constants.py

```diff
@@ -72,16 +82,34 @@
 so copy its behavior.
 """
 
+METADATA_PATH = Path("/etc/podinfo")
+"""Default path to injected pod metadata."""
+
+MOUNT_PATH_DOWNWARD_API = "/opt/lsst/software/jupyterlab/runtime"
```

Comment:
I feel like we ought to have the main configurable as some name for /opt/lsst/software and then tack things on to that.  (and maybe make it more generic at the same time)

Reply:
Yes, that would make sense. I'll add that to my list of things to do when we make it configurable. I'm hoping we can also get rid of a few of the things we're mounting. Both the environment variables and the downward API in the user lab are of questionable utility.